
/usr/lib/python2.7/dist-packages/rpy2/rinterface/__init__.py:186: RRuntimeWarning: Loading required package: rngWELL

  warnings.warn(x, RRuntimeWarning)
/usr/lib/python2.7/dist-packages/rpy2/rinterface/__init__.py:186: RRuntimeWarning: This is randtoolbox. For an overview, type 'help("randtoolbox")'.

  warnings.warn(x, RRuntimeWarning)
expected=14981109.589145
actual=14981109.589145
diff=0.000000
diff=1
expected=14981109.589145
actual=14981109.589145
diff=0.000000
diff=1
expected=14981109.589145
actual=14981109.589145
diff=0.000000
diff=1
expected=14981109.589145
actual=14981109.589145
diff=0.000000
diff=1
expected=14981109.589145
actual=14981109.589145
diff=0.000000
diff=1
expected=14981109.589145
actual=14981109.589145
diff=0.000000
diff=1
expected=14981109.589145
actual=14981109.589145
diff=0.000000
diff=1
expected=14981109.589145
actual=14981109.589145
diff=0.000000
diff=1
expected=14981109.589145
actual=14981109.589145
diff=0.000000
diff=1
expected=14981109.589145
actual=14981109.589145
diff=0.000000
diff=1
expected=14981109.589145
actual=14981109.589145
diff=0.000000
diff=1
expected=14981109.589145
actual=14981109.589145
diff=0.000000
diff=1
expected=14981109.589145
actual=14981109.589145
diff=0.000000
diff=1
expected=14981109.589145
actual=14981109.589145
diff=0.000000
diff=1
expected=14981109.589145
actual=14981109.589145
diff=0.000000
diff=1
expected=14981109.589145
actual=14981109.589145
diff=0.000000
diff=1
expected=14981109.589145
actual=14981109.589145
diff=0.000000
diff=1
expected=14981109.589145
actual=14981109.589145
diff=0.000000
diff=1
expected=14981109.589145
actual=14981109.589145
diff=0.000000
diff=1
expected=14981109.589145
actual=14981109.589145
diff=0.000000
diff=1
expected=14981109.589145
actual=14981109.589145
diff=0.000000
diff=1
expected=14981109.589145
actual=14981109.589145
diff=0.000000
diff=1
expected=14981109.589145
actual=14981109.589145
diff=0.000000
diff=1
expected=14981109.589145
actual=14981109.589145
diff=0.000000
diff=1
expected=14981109.589145
actual=14981109.589145
diff=0.000000
diff=1
expected=14981109.589145
actual=14981109.589145
diff=0.000000
diff=1
/usr/lib/python2.7/dist-packages/rpy2/rinterface/__init__.py:186: RRuntimeWarning: Loading required package: foreach

  warnings.warn(x, RRuntimeWarning)
/usr/lib/python2.7/dist-packages/rpy2/rinterface/__init__.py:186: RRuntimeWarning: Loading required package: iterators

  warnings.warn(x, RRuntimeWarning)
/usr/lib/python2.7/dist-packages/rpy2/rinterface/__init__.py:186: RRuntimeWarning: Loading required package: parallel

  warnings.warn(x, RRuntimeWarning)
expected=14981109.589145
actual=14981109.589145
diff=0.000000
diff=1
expected=14981109.589145
actual=14981109.589145
diff=0.000000
diff=1
expected=14981109.589145
actual=14981109.589145
diff=0.000000
diff=1
expected=14981109.589145
actual=14981109.589145
diff=0.000000
diff=1
expected=14981109.589145
actual=14981109.589145
diff=0.000000
diff=1
expected=14981109.589145
actual=14981109.589145
diff=0.000000
diff=1
expected=14981109.589145
actual=14981109.589145
diff=0.000000
diff=1
/usr/lib/python2.7/dist-packages/rpy2/rinterface/__init__.py:186: RRuntimeWarning: Error in chol.default(R) : 
  the leading minor of order 30 is not positive definite

  warnings.warn(x, RRuntimeWarning)
/usr/lib/python2.7/dist-packages/rpy2/rinterface/__init__.py:186: RRuntimeWarning: In addition: 
  warnings.warn(x, RRuntimeWarning)
/usr/lib/python2.7/dist-packages/rpy2/rinterface/__init__.py:186: RRuntimeWarning: There were 26 warnings (use warnings() to see them)
  warnings.warn(x, RRuntimeWarning)
/usr/lib/python2.7/dist-packages/rpy2/rinterface/__init__.py:186: RRuntimeWarning: 

  warnings.warn(x, RRuntimeWarning)
Traceback (most recent call last):
  File "../../../orcc", line 27, in <module>
    orio.main.orio_main.start(sys.argv, orio.main.orio_main.C_CPP)
  File "/root/dlmt_spapt_experiments/orio/orio/main/orio_main.py", line 109, in start
    optimized_code_seq = odriver.optimizeCodeFrags(cfrags, True)
  File "/root/dlmt_spapt_experiments/orio/orio/main/opt_driver.py", line 45, in optimizeCodeFrags
    cur_seq = self.__optimizeCodeFrag(cf, perf_params)
  File "/root/dlmt_spapt_experiments/orio/orio/main/opt_driver.py", line 120, in __optimizeCodeFrag
    cfrag.cfrags)
  File "/root/dlmt_spapt_experiments/orio/orio/main/tuner/tuner.py", line 152, in tune
    best_perf_params, best_perf_cost = search_eng.search()
  File "/root/dlmt_spapt_experiments/orio/orio/main/tuner/search/search.py", line 133, in search
    best_coord,best_perf,search_time,speedup = self.searchBestCoord(startCoord)
  File "/root/dlmt_spapt_experiments/orio/orio/main/tuner/search/gpr/gpr.py", line 548, in searchBestCoord
    best_point, used_points = self.gpr()
  File "/root/dlmt_spapt_experiments/orio/orio/main/tuner/search/gpr/gpr.py", line 513, in gpr
    best_predicted_points = robjects.r(r_snippet)
  File "/usr/lib/python2.7/dist-packages/rpy2/robjects/__init__.py", line 362, in __call__
    res = self.eval(p)
  File "/usr/lib/python2.7/dist-packages/rpy2/robjects/functions.py", line 178, in __call__
    return super(SignatureTranslatedFunction, self).__call__(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/rpy2/robjects/functions.py", line 106, in __call__
    res = super(Function, self).__call__(*new_args, **new_kwargs)
rpy2.rinterface.RRuntimeError: Error in chol.default(R) : 
  the leading minor of order 30 is not positive definite

